README.md

```diff
@@ -387,7 +388,11 @@ In this test case, `@select() counter$: Observable<number>;` is used to get the
 
 ### ngrx [:top:](#current-status)
 
-Setting up basic `ng2-redux` is fine with AoT. But similar to `ng2-redux-@select`, `ngrx/Effects` does not work with `@ngtools/webpack`. Details can be found here: https://github.com/angular-redux/ng2-redux/issues/236.
+Setting up basic `ngrx` is fine with AoT.
+
+### ngrx-effects [:top:](#current-status)
+
+Setting up basic `ngrx-effects` is fine with AoT. But similar to `ng2-redux-@select`, `ngrx/Effects` does not work with `@ngtools/webpack`, due to custom decorators being stripped. Details can be found here: https://github.com/angular-redux/ng2-redux/issues/236.
 
```

Comment:
Could you also include this link: https://github.com/angular/angular-cli/issues/2799 ? Thanks